src/main/scala/org/apache/spark/ml/linalg/RAPIDSML.scala

```diff
@@ -73,6 +74,34 @@ private[spark] object RAPIDSML extends Serializable {
     jniRAPIDSML.dgemm(transa, transb, m, n, k, alpha, A.values, lda, B.values, ldb, beta, C.values, ldc, deviceID)
   }
 
+  /**
+   * Wrapper of cuBLAS GEMM routine. Most parameters are the same as: https://docs.nvidia.com/cuda/cublas/index.html#cublas-lt-t-gt-gemm
+   * but modified B and removed C for integration compatibility.
+   * B is a long value representing the handle of a `cudf::column_view *` which is holding the matrix data in device.
+   * C as the computation output is removed from parameter list and becomes a return value of Long type that represents
+   * the handle of a LIST type `cudf::column_view *` which is holding the computation output.
+   * @param transa CublasOperationT enum value representing the operation op(A) that is non- or (conj.) transpose.
+   * @param transb CublasOperationT enum value representing the operation op(B) that is non- or (conj.) transpose.
+   * @param m number of rows of matrix op(A) and C.
+   * @param n number of columns of matrix op(B) and C.
+   * @param k number of columns of op(A) and rows of op(B).
+   * @param alpha scalar used for multiplication.
+   * @param A array of dimensions lda x k with lda>=max(1,m) if transa == CUBLAS_OP_N and lda x m with lda>=max(1,k) otherwise.
+   * @param lda leading dimension of two-dimensional array used to store the matrix A.
+   * @param B long value representing the handle of a `cudf::column_view *` which is holding the matrix data in device.
+   *          (Array of dimension ldb x n with ldb>=max(1,k) if transb == CUBLAS_OP_N and ldb x k with ldb>=max(1,n) otherwise.)
```

Comment:
Rather than take such a "raw" interface as a `cudf::column_view *`, this should take a `ColumnView` for `B`.

Reply:
Updated, it's now taking ColumnView as a parameter.